tests/rbac_test.go

```diff
@@ -0,0 +1,213 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ktests "kubevirt.io/kubevirt-ansible/tests"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/log"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("RBAC", func() {
+	flag.Parse()
+
+	var vmi ktests.VMManifest
+
+	vmi.Name = "vm-cirros"
+	vmi.Manifest = "tests/manifests/vm-cirros.yaml"
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	BeforeEach(func() {
+		By(fmt.Sprintf("Login with the user %s", ktests.UsernameAdminUser))
+		args := []string{"login", "-u", ktests.UsernameAdminUser, "-p", "123456"}
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		tests.BeforeTestCleanup()
+		ktests.CreateUser()
+
+		By("Create a VM with yaml.")
+		args = []string{"create", "-f", vmi.Manifest, "-n", tests.NamespaceTestAlternative}
+		_, err = tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Check VMI should create.")
+		args = []string{"start", vmi.Name, "-n", tests.NamespaceTestAlternative}
+		_, err = tests.RunCommand("virtctl", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Check the vmi is running.")
+		args = []string{"get", "vmi", vmi.Name, "--template", "{{.status.phase}}", "-n", tests.NamespaceTestAlternative}
+		Eventually(func() string {
+			output, err := tests.RunCommand("oc", args...)
+			Expect(err).ToNot(HaveOccurred())
+			return output
+		}, time.Minute*10).Should(Equal("Running"))
+	})
+
+	AfterEach(func() {
+		By(fmt.Sprintf("Login with the user %s", ktests.UsernameAdminUser))
+		args := []string{"login", "-u", ktests.UsernameAdminUser, "-p", "123456"}
+		_, err := tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		ktests.DeleteUser()
+
+		By(fmt.Sprintf("Delete identity of user %s", ktests.UsernameTestUser))
+		args = []string{"delete", "identity", "allow_all_auth:"+ktests.UsernameTestUser}
+		_, err = tests.RunCommand("oc", args...)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	DescribeTable("RBAC Authorization", func(role, namespace string, con bool) {
```

Comment:
what is `con` stands for?

Reply:
I guess it means `condition`.

Reply:
con means a condition, for a user, that it will able to create resources or not with the available permission.